backend/clickhouse/clickhouse.go

```diff
@@ -37,29 +41,37 @@ func GetPostgresConnectionString() string {
 }
 
 func NewClient(dbName string) (*Client, error) {
-	opts := getClickhouseOptions(dbName)
+	opts := getClickhouseOptions(dbName, DefaultUser)
 	opts.MaxIdleConns = 10
 	opts.MaxOpenConns = 100
-
 	conn, err := clickhouse.Open(opts)
 
+	optsReadonly := getClickhouseOptions(dbName, ReadonlyUser)
+	optsReadonly.MaxIdleConns = 10
+	optsReadonly.MaxOpenConns = 100
+	connReadonly, errReadonly := clickhouse.Open(optsReadonly)
+
 	go func() {
 		for {
-			stats := conn.Stats()
-			log.WithContext(context.Background()).WithField("Open", stats.Open).WithField("Idle", stats.Idle).WithField("MaxOpenConns", stats.MaxOpenConns).WithField("MaxIdleConns", stats.MaxIdleConns).Debug("Clickhouse Connection Stats")
-			hmetric.Gauge(context.Background(), "clickhouse.open", float64(stats.Open), nil, 1)
-			hmetric.Gauge(context.Background(), "clickhouse.idle", float64(stats.Idle), nil, 1)
+			for _, c := range []driver.Conn{conn, connReadonly} {
+				stats := c.Stats()
+				name := lo.Ternary(c == conn, "conn", "connReadonly")
+				log.WithContext(context.Background()).WithField("Open", stats.Open).WithField("Idle", stats.Idle).WithField("MaxOpenConns", stats.MaxOpenConns).WithField("MaxIdleConns", stats.MaxIdleConns).Debugf("Clickhouse Connection Stats - %s", name)
+				hmetric.Gauge(context.Background(), "clickhouse.open", float64(stats.Open), nil, 1)
+				hmetric.Gauge(context.Background(), "clickhouse.idle", float64(stats.Idle), nil, 1)
+			}
```

Comment:
could send a tag here
```suggestion
tags := []attribute.KeyValue{attribute.String("name", name)}
				hmetric.Gauge(context.Background(), "clickhouse.open", float64(stats.Open), tags, 1)
				hmetric.Gauge(context.Background(), "clickhouse.idle", float64(stats.Idle), tags, 1)
			}
```